packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/plugin-ledger-connector-fabric.ts

```diff
@@ -103,6 +104,13 @@ import {
   CertDatastore,
   IIdentityData,
 } from "./identity/internal/cert-datastore";
+import { GetTransactionReceiptByTxIDEndpointV1 } from "./get-transaction-receipt/get-transaction-receipt-by-txid-endpoint-v1";
+//import { common } from "fabric-protos";
```

Comment:
@jscode017 If not used, please remove the commented imports.

Reply:
Ok, resolve with new commit